timeseries/src/autogluon/timeseries/models/presets.py

```diff
@@ -75,6 +81,9 @@
     TemporalFusionTransformerMXNet=50,
     SimpleFeedForward=40,
     TransformerMXNet=30,
+    AutoARIMA=30,
+    AutoETS=70,
+    DynamicOptimizedTheta=60,
```

Comment:
How do we determine the priorities of these stats models? How will they added to existing ARIMA and ETA impl? Some accuracy metrics comparison might be helpful to give insights with ensemble results

Reply:
I set the priorities inversely proportional to the average time it takes to fit these models (slower model -> lower priority).

Reply:
Also added models to the presets based on the discussion below.